services/exonum-time/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "exonum-time"
+version = "0.5.0"
+authors = ["The Exonum Team <exonum@bitfury.com>"]
+homepage = "https://github.com/exonum/exonum-time"
+repository = "https://github.com/exonum/exonum-time"
+documentation = "https://docs.rs/exonum-time"
+readme = "README.md"
+license = "Apache-2.0"
+keywords = ["exonum", "time"]
+categories = ["cryptography", "date-and-time"]
+description = "The time oracle service for Exonum."
+
+[badges]
+travis-ci = { repository = "exonum/exonum" }
+
+[dependencies]
+exonum = { version = "0.5.1", path = "../../exonum" }
+iron = "0.6.0"
+bodyparser = "0.8.0"
+router = "0.6.0"
+serde = "1.0"
+serde_json = "1.0"
+serde_derive = "1.0"
+pretty_assertions = "0.4.0"
```

Comment:
Just noticed: this is an ordinary dependency? Where is it used? 🤔

Reply:
`pretty_assertions` is a [special library](https://github.com/colin-kiegel/rust-pretty-assertions): it should not be used directly, but it makes the assertions output nicer. Honestly, I'm not sure if we need that, so other opinions are welcome.

Perhaps, it makes sense to add it as `[dev-dependencies]`. In that case, it should be done for all our crates.